driver/network/local/local.go

```diff
@@ -76,6 +76,10 @@ type LocalNetwork struct {
 	rpcWorkerPool *rpc.RpcWorkerPool
 }
 
+var LocalVolumes = map[string]string{
+	"ipc": "/datadir/ipc",
```

Comment:
Is this the path in docker or host? It looks like something on docker. I'm asking because in `driver/docker/docker.go`, the term `local` seems to refer to a path in host.

Reply:
The format is `localname:/path/in/docker`, where `localname` is the name of volume on the host.
This `localname` can also be expanded so that it accepts `localpath`, which is path on host.
For now this is enough for testing/checking status from ipc, so `localpath` can be added as a separate PR.